bundles/org.eclipse.passage.lic.features.model/plugin.xml

```diff
@@ -22,4 +22,11 @@
             genModel="model/features.genmodel"/>
    </extension>
 
+   <extension point="org.eclipse.emf.ecore.extension_parser">
+      <!-- @generated features -->
+      <parser
+            type="features"
```

Comment:
should it correspond to `.features_xmi` file extension?

Reply:
Good point!